transaction-exclusion-api/app/src/integrationTest/kotlin/net/consensys/linea/transactionexclusion/TransactionExclusionAppTest.kt

```diff
@@ -174,19 +177,19 @@ class TransactionExclusionAppTest : CleanDbTestSuiteParallel() {
       "jsonrpc": "2.0",
       "id": 125,
       "method": "linea_getTransactionExclusionStatusV1",
-      "params": ["0x526e56101cf39c1e717cef9cedf6fdddb42684711abda35bae51136dbb350ad7"]
+      "params": ["${defaultRejectedTransaction.transactionInfo.hash.encodeHex()}"]
     }
     """.trimIndent()
 
-    // Check the get response is corresponding to the rejected txn from SEQUENCER
+    // Check the get response is corresponding to the rejected tx from SEQUENCER
     assertThatJson(makeRequestJsonResponse(getTxJsonRequest))
       .isEqualTo(
         """{
           "jsonrpc": "2.0",
           "id": 125,
           "result": {
-            "txHash": "0x526e56101cf39c1e717cef9cedf6fdddb42684711abda35bae51136dbb350ad7",
-            "from": "0x4d144d7b9c96b26361d6ac74dd1d8267edca4fc2",
+            "txHash": "${defaultRejectedTransaction.transactionInfo.hash.encodeHex()}",
+            "from": "${defaultRejectedTransaction.transactionInfo.from.encodeHex()}",
```

Comment:
I know this is to avoid duplication, but this case duplication here helped to read the test better. PLease see [DAMP vs DRY](https://abseil.io/resources/swe-book/html/ch12.html#tests_and_code_sharing_dampcomma_not_dr)

In the sample below, I can't see how monces are serialized, numeric or hexadecimal?